p2p/libp2p/options.go

```diff
@@ -0,0 +1,21 @@
+package libp2p
+
+import (
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+// WithPeerBlackList defines the option of setting a peer black list handler
+func WithPeerBlackList(blacklistHandler p2p.BlacklistHandler) p2p.Option {
+	return func(cfg *p2p.Config) error {
```

Comment:
is p2p.Option implementing error ?

Reply:
no, Option is just a type of a function pointer